react-native/react/actions/tracker.js

```diff
@@ -84,10 +86,40 @@ export function onFollowHelp (username: string): Action {
   }
 }
 
-export function onCloseFromActionBar (username: string): Action {
-  return {
-    type: Constants.onCloseFromActionBar,
-    payload: {username}
+function trackUser (username: string, state: {tracker: RootTrackerState}): Promise<boolean> {
```

Comment:
but ultimately that could be captured in the store state right? i don't think the action that triggers it necessarily cares